bin/video2x.json

```diff
@@ -17,12 +19,13 @@
     "tta": 0,
     "input_path": null,
     "output_path": null,
-    "model_dir": null,
+    "model_dir": "models/cunet",

```

Comment:
That _is_ the default waifu2x-caffe value these days. I think this is a good change for clarity sake. Though, word to the wise, CUnet is also the most processor-intensive.

The settings file would really benefit from comments to help supplement documentation, which is something that JSON does not typically allow. I wouldn't mind submitting a PR to convert all of the JSON to some flavor of version-controlled YAML.

Reply:
We already have documentation for the JSON file.
https://github.com/K4YT3X/video2x/wiki/Basic-Configuration

Reply:
Well I did say supplement. More people read in-line documentation since there is no cross-referencing required.

Reply:
btw, for anyone curios, this change happened because find_path can take "model/cunet" and search for the proper path, if needed.  find_path can not do that for null.